app/Resources/views/blog/rss.xml.twig

```diff
@@ -0,0 +1,23 @@
+{% spaceless %}
```

Comment:
Just for remove space between tags, but this detail can be ignored for simplicity :+1: Removed!